packages/amplify-cli-core/src/plugin-facade/api-category-facade.ts

```diff
@@ -26,7 +24,7 @@ export class ApiCategoryFacade {
    * Perform the actual transformation for a given project. This is predominantlyu a side-effecting call, but we
    * also return the deployment resources as well.
    */
-  static async transformGraphQLSchema(context: $TSContext, options: $TSAny): Promise<DeploymentResources | undefined> {
+  static async transformGraphQLSchema(context: $TSContext, options: $TSAny): Promise<$TSAny | undefined> {
```

Comment:
Why?

Reply:
Type is no longer exported from our core packages, and this type isn't actually used anywhere in the system. I could import the type directly, but instead elected to just leave this as an `any` which was the state before we added these facades.